base/pkg/entry.jl

```diff
@@ -11,6 +13,7 @@ macro recover(ex)
         try $(esc(ex))
         catch err
             show(err)
+            print('\n')
```

Comment:
`show` doesn't print new line, so I added it, otherwise an output on a screen is mangled.